tests/413.2-envelopes.js

```diff
@@ -18,26 +18,36 @@ const should = chai.should();
 
 const require = createRequire(import.meta.url);
 
-const tag = 'EnvelopingProof';
+const tag = 'vc2.0';
 const {match} = filterByTag({tags: [tag]});
 
 // 4.12.1 Enveloped Verifiable Credentials https://w3c.github.io/vc-data-model/#enveloped-verifiable-credentials
 describe('VP - Enveloped Verifiable Credentials', function() {
   setupMatrix.call(this, match);
   for(const [name, implementation] of match) {
     const endpoints = new TestEndpoints({implementation, tag});
+    const issuer_envelope_support =
+      endpoints.issuer.settings.tags.includes(
+        'EnvelopingProof');
+    const vpVerifier_envelope_support =
+      endpoints.vpVerifier.settings.tags.includes(
+        'EnvelopingProof');
 
     describe(name, function() {
       let issuedVc;
       before(async function() {
-        try {
-          issuedVc = await endpoints.issue(require(
-            './input/credential-ok.json'));
-        } catch(e) {
-          console.error(
-            `Issuer: ${name} failed to issue "credential-ok.json".`,
-            e
-          );
+        if(issuer_envelope_support) {
```

Comment:
We don't usually use "snake_case", fwiw. It's a nit, I realize, but probably best to be consistent.

Reply:
my mistake, I was wearing my python eye wear ;)